r/R/dplyr-funcs-datetime.R

```diff
@@ -211,9 +212,15 @@ register_bindings_datetime_components <- function() {
     hour <- Expression$create("hour", x)
     hour < 12
   })
+
   register_binding("pm", function(x) {
     !call_binding("am", x)
   })
+
+  register_binding("leap_year", function(date) {
+    Expression$create("is_leap_year", date)
+  })
+
```

Comment:
I could very well be very wrong, but I think this is handled by [expression.R](https://github.com/apache/arrow/blob/master/r/R/expression.R#L78) and can removed completely. (Bad rebase, mea culpa).

Reply:
```suggestion
```

Reply:
Thanks! I hadn't spotted that!